lambda_functions/user_service/create_user.py

```diff
@@ -29,13 +29,16 @@ def create_user(event):
                 GroupName = 'Detective'
             )
 
-            # Add mail address if submitted
-            if 'email' in event['request']['userAttributes']:
+            # Add mail address if submitted and set confirmation status
+            if 'custom:mail_subscription' in event['request']['userAttributes']:
+                confirmation_status = event['request']['userAttributes']['custom:mail_subscription']
                 mail = Mail()
                 mail.email = event['request']['userAttributes']['email']
                 mail.user_id = user.id
+                if confirmation_status == 0:
```

Comment:
Der confirmation_status ist doch ein String. Siehe das Beispiel-Objekt, das ich dir geschickt habe.
Sorry - Das hatte ich dir falsch gesagt. Ich konfiguriere das zwar in Cognito als "number" aber irgendwie wird das trotzdem als String verschickt.